tests/unit/util/test_getter.py

```diff
@@ -311,6 +306,16 @@ def test_parses_content(self, method_name, input_content, expected_output):
 
 
 class TestHttpGetter:
+    def teardown_method(self):
+        if "LOGPREP_CONFIG_AUTH_METHOD" in os.environ:
+            del os.environ["LOGPREP_CONFIG_AUTH_METHOD"]
+        if "LOGPREP_CONFIG_AUTH_TOKEN" in os.environ:
+            del os.environ["LOGPREP_CONFIG_AUTH_TOKEN"]
+        if "LOGPREP_CONFIG_AUTH_USERNAME" in os.environ:
+            del os.environ["LOGPREP_CONFIG_AUTH_USERNAME"]
+        if "LOGPREP_CONFIG_AUTH_PASSWORD" in os.environ:
+            del os.environ["LOGPREP_CONFIG_AUTH_PASSWORD"]
+
```

Comment:
similar to what I described above. With a `with`-context and mocking the envs this code shouldn't be necessary.

Reply:
done